src/cap-description.adoc

```diff
@@ -137,6 +137,9 @@ If a capability grants <<r_perm>> and <<c_perm>>, but no <<lm_perm>>, then a cap
 The rules specified by <<ACPERM>> are followed when <<w_perm>> and <<lm_perm>> are removed, so additional permissions may also be removed.
 Clearing a capability's <<lm_perm>> and <<w_perm>> allows sharing a read-only version of a data structure (e.g. a tree or linked list) without making a copy.
 
+NOTE: Implementations are allowed to propagate, but not alter, reserved permissions when performing <<lm_perm>> check on a loaded capability.
+ This simplifies the number of cases handled by the implementation as it's a choice whether to remove _all_ permissions.
```

Comment:
Isn't this already covered as part of this statement?
> loaded capabilities that are sealed or untagged do not have their permissions changed

I would expect a capability that has "reserved" or "invalid" permissions would be untagged because of the way CHERI RISC-V is defined. I guess the strange case where this could happen is if there is some fault/bit-flip and the capability ended up tagged with reserved permissions, which would be very awkward, but ideally it would be detected as an error.

Reply:
it's exactly the case you mentioned - what happens if we get an illegal cap -tagged with a reserved perm value.
certainly these should be untagged in an ideal world, but we need to handle the unexpected cases.

Reply:
Ok, that case is a little concerning. What damage could a capability that is tagged, but invalid in some way, cause? For example, would a LC succeed if it was provided a capability with "reserved" permissions but somehow has R and C set?

At the very least, it would be good to amend the note here to clearly state the case we are concerned about.

Reply:
optional divergence from ACPERM for tagged caps with reserved permissions, this is a don't care state and so it's important to have the option to simplify the hardware

Reply:
I was a bit confused by this wording, but if it's just about invalid capabilities loaded from memory due to an error, maybe this could be simplified to something like "unlike  explicit <<ACPERM>>, implemementations are permitted to retain invalid combinations loaded from memory instead of canonicalizing to _no permissions_"?

Reply:
ok - try again - I had a look and `canonicalizing` isn't currently in the spec so I avoided using it.